tests/testthat/test-param-labels.R

```diff
@@ -101,26 +98,26 @@ withr::with_options(list(bbr.bbi_exe_path = read_bbi_path()), {
 
       on.exit({ if (fs::file_exists(file.path(PL_MODEL_DIR, glue("{.mod_id}.yaml")))) fs::file_delete(file.path(PL_MODEL_DIR, glue("{.mod_id}.yaml"))) })
 
-      .param_df <- .mod %>% bbr::model_summary() %>% param_estimates()
+      .param_df <- .mod %>% model_summary() %>% param_estimates()
       names(.param_df) <- names(.param_df) %>% tolower()
 
       # make label df
       .label_df <- .mod %>% param_labels()
 
       # join to constuct full parameter table
       suppressSpecificWarning({
-        .new_df <- inner_join(.param_df,
-                              .label_df %>% apply_indices(.omega = MODEL_PICK$omega, .sigma = MODEL_PICK$sigma) %>% select(-param_type),
+        .new_df <- dplyr::inner_join(.param_df,
+                              .label_df %>% apply_indices(.omega = MODEL_PICK$omega, .sigma = MODEL_PICK$sigma) %>% dplyr::select(-param_type),
                               by = SUMMARY_PARAM_NAMES)
       }, .regexpr = "Column .+ has different attributes on LHS and RHS of join")
 
       # join against reference to make sure they're the same
-      ref_df <- ref_df %>% mutate(!!SUMMARY_PARAM_NAMES := ifelse(param == "THETA",
+      ref_df <- ref_df %>% dplyr::mutate(!!SUMMARY_PARAM_NAMES := ifelse(param == "THETA",
```

Comment:
Not something that needs to be coupled with this clean-up (and perhaps not something to bother with at all?), but just a note for completeness:  the tests include other unqualified `mutate`s as well as `select`s and `filter`s that make it through without an error because they happen to be in NAMESPACE.